### DISCUSSION TITLE
CLI client stops updating after 24 hours?

### DISCUSSION THREAD

**Discussion by gprime31** [projectdiscovery/interactsh#179] [Q&A] [answered]
CLI client stops updating after 15-24 hours with fuff running.

**Comment by ehsandeep** [comment, marked as answer]
that's true and expected, currently server is running with eviction of 1 day, so any payload will be deregister after 24 hours due to single server in use, this will change after next release where interactsh client will be using pool of interactsh server - https://github.com/projectdiscovery/interactsh/pull/177

**Reply by gprime31** [reply]
oh ok
thanks
